CMakeLists.txt

```diff
@@ -85,6 +87,19 @@ add_compile_options(/WX /Gy
     # note that /Zi generates debug info inside the object file, it's the same as /Z7 for msvc
     "$<$<COMPILE_LANGUAGE:ASM_MASM>:/Zi;/W3;/nologo>")
 
+if(STL_USE_ANALYZE)
+    # warning C6553: The annotation for function 'LCMapStringEx' on _Param_(9)
+    # does not apply to a value type.
+    # There's a bug in the declaration for LCMapStringEx - it applies _In_opt_ to an LPARAM.

```

Comment:
Even if we don't have an OS bug to cite, we should mention what version of the WinSDK this occurs in. This will allow someone in the future to notice that the comment is old, and check whether things have changed.

Additionally, temporarily removing this suppression and passing `-k 1000` to Ninja (keep going until 1000 errors) reveals that there are more occurrences throughout the WinSDK. We should mention that here (although we don't need to cite all of them).